content/reference/admin/private_locations/configuration/kubernetes/index.md

```diff
@@ -48,7 +48,7 @@ control-plane {
       # They are hosted on Docker Hub, and available for the linux/amd64 and linux/arm64 platforms
       image {
         type = certified
-        java = latest # Possible values : 8, 11, 17 or latest
+        java = latest # Possible values : 8, 11, 17, 21 or latest
```

Comment:
Need to update and release https://github.com/gatling/frontline-cloud/blob/dev/back/commons/primitive/src/main/scala/io/gatling/enterprise/commons/primitive/CertifiedDockerImage.scala#L12 first

Reply:
oh right